packages/enhanced/src/lib/container/ContainerEntryModule.ts

```diff
@@ -96,7 +96,7 @@ class ContainerEntryModule extends Module {
   override identifier(): string {
     return `container entry (${this._shareScope}) ${JSON.stringify(
       this._exposes,
-    )}`;
+    )} ${this._injectRuntimeEntry}`;
```

Comment:
needed to allow multiple container entry dependency to be created, otherwise the factory uses cache when changing federation entry module types.